packages/webcomponents/src/api/TerminalOrder.ts

```diff
@@ -0,0 +1,68 @@
+export interface ITerminalOrder {
+  businessId: string;
+  accountId: string;
+  provider: string;
+  orderType: string;
+  orderItems: iTerminalItem[];
+}
+
+export interface iTerminalItem {
+  modelName: string;
+  quantity: number;
+}
+
+export class TerminalOrder implements ITerminalOrder {
+  public businessId: string;
+  public accountId: string;
+  public provider: string = 'verifone';
+  public orderType: string;
+  public orderItems: iTerminalItem[];
+
+  constructor(terminalOrder?: ITerminalOrder) {
+    this.businessId = terminalOrder?.businessId || '';
+    this.accountId = terminalOrder?.accountId || '';
+    this.provider = terminalOrder?.provider || '';
+    this.orderType = terminalOrder?.orderType || '';
+    this.orderItems = terminalOrder?.orderItems || [];
+  }
+
+  public get payload() {
```

Comment:
Can we avoid this if we just keep the naming convention 1:1 with the backend? Keep it all snake case? Having this getter is helpful but I think going this way is prone to confusion and naming typos